qiita_pet/templates/user_messages.html

```diff
@@ -0,0 +1,10 @@
+{% extends sitebase.html %}
+{% autoescape None %}
+{% block content%}
+<h2 style="text-decoration: underline;">System Messages</h2>
+<ul>
```

Comment:
Ah, thought you meant just the line you commented on as that's all I had to go on. 

Autoescape None is set, so it will always print raw on this page. Makes sense considering what this page is.